### ISSUE TITLE
StackOverflowException during registration

### ISSUE THREAD

**Issue by ennki** [haison8x/MediatR.SimpleInjector#1]
When call BuildMediator extension method like this:

`container.BuildMediator(typeof(TestCommand).GetTypeInfo().Assembly);`

 it always call itself (version with params), and eventually raise StackOverflowException.

**Comment by ennki** [comment]
Can be fixed next way:

```
public static Container BuildMediator(this Container container, params Assembly[] assemblies)
{
    return BuildMediator(container, (IEnumerable<Assembly>)assemblies);
}
```

**Comment by haison8x** [comment]
I fixed this issue in version 1.0.0.2. Thanks @ennki
